src/js/api/build.js

```diff
@@ -11,6 +11,50 @@ if (
     var ripe = base.ripe;
 }
 
+/**
+ * Gets the existing builds, according to the provided filtering
+ * strategy as normalized values.
+ *
+ * @param {Object} options An object of options to configure the request, such as:
+ * - 'filters[]' - List of filters that the query will use to, operators such as
+ * ('in', 'not_in', 'like', 'contains'), for instance (eg: 'id:eq:42') would filter by the id that equals to 42.
+ * - 'sort' - List of arguments to sort the results by and which direction
+ * to sort them in (eg: 'id:ascending') would sort by the id attribute in ascending order,
+ * while (eg: 'id:descending')] would do it in descending order.
+ * - 'skip' - The number of the first record to retrieve from the results.
+ * - 'limit' - The number of results to retrieve.
+ * @param {Function} callback Function with the result of the request.
+ * @param {Function} callback Function with the result of the request.
+ * @returns {XMLHttpRequest} The XMLHttpRequest instance of the API request.
+ */
+ripe.Ripe.prototype.getBuilds = function(options, callback) {

```

Comment:
@3rdvision actually we don't support all these options. Unfortunately, the build listing endpoint is much dumber (see https://github.com/ripe-tech/ripe-core/blob/89e4b47edf05120e33e59c8c86bb20568979eb80/src/ripe_core/controllers/api/build.py#L20). So we need to remove that from here.